js/loadTrack.js

```diff
@@ -0,0 +1,24 @@
+$('.trackLink').click(function(e) {
+    var link = e.target.dataset.track
+        
+    if  (link === '1' && $('#track1').length === 0) {
```

Comment:
This large if statement is essentially repeated below. 
- abstract out into a function that can be called to save repeating yourselves

Reply:
done